tests/test_custom_models.py

```diff
@@ -865,7 +884,8 @@ def test_disable_adapters(self, test_name, model_id, config_cls, config_kwargs):
         )
         model = get_peft_model(model, config)
         model.eval()
-        outputs_before = model(**X)
+        with model.disable_adapter():
```

Comment:
Why make this change? It shouldn't be required.

Reply:
We find that when you use `model = get_peft_model(model, config)`, there would be an adapter called 'default' being activated. And it would change the model output even if it is not trained (not zero initialization).

Reply:
I see. So I think the issue is that FourierFT cannot be initialized so that it would be an identity transform, is that right? E.g. for LoRA, the `lora_B` weight is initialized as zeros, so that by default, at the start of training, LoRA is an identity transform. Would it be possible to have such an init option for FourierFT? If not, I would recommend to add a check to this test and to skip it if FourierFT is being used. But please don't edit the test for all other methods.

Reply:
Thanks for the comments. I misunderstood the idea of this test. I thought it was to see if the adapters are correctly disabled. In fact, we can set `spectrum` to zeros to make it an Identity mapping. We will revise it.

Reply:
This change still needs to be reverted.